lib/process/process.go

```diff
@@ -2091,7 +2091,15 @@ func (p *Process) loginWithToken(tokenID string, w http.ResponseWriter, r *http.
 	}
 }
 
-func (p *Process) loadRPCCredentials() (*rpcserver.Credentials, utils.TLSArchive, error) {
+func (p *Process) loadRPCCredentials() (creds *rpcserver.Credentials, archive utils.TLSArchive, err error) {
+	err = utils.Retry(defaults.RetryInterval, defaults.RetryAttempts, func() error {
```

Comment:
I'd be cautious about running anything in a loop for **any** error - is there a specific set of errors this can be retried for?